app/services/FaciaPress.scala

```diff
@@ -82,6 +116,14 @@ class FaciaPress(val faciaPressQueue: FaciaPressQueue, val configAgent: ConfigAg
             case Success(_) =>
               EnqueuePressSuccess.increment()
           }
+          val fut2 = Future.traverse(paths)(path => faciaPressTopic.publish(PressJob(FrontPath(path), Draft, forceConfigUpdate = pressCommand.forceConfigUpdate)))
+          fut2.onComplete {
+            case Failure(error) =>
+              EnqueuePressFailure.increment()
+              logger.error("Error manually pressing draft collection through update from tool", error)
+            case Success(_) =>
+              EnqueuePressSuccess.increment()
+          }
```

Comment:
The `fut` and `fut2` variables only need to co-exist while this is not implemented on the frontend side. Once the frontend queue is subscribed to the topic, only the second variable will be required.

Reply:
Makes sense. Not blocking, but I wonder if there's a better name that can be given than `fut` (and later `fut2`) which doesn't feel very descriptive to me. If I'm understanding correctly it provides the outcome of a "send message" or "publish" action?

Reply:
Yes, I was thinking once subsequent changes are made and one of the variables is removed, a better name should be given to it.